scripts/monitoring/ops-metric-client.py

```diff
@@ -64,6 +65,21 @@ def run(self):
 
         self.metric_sender.send_metrics()
 
+    @staticmethod
+    def adjust_type(val, metric_type):
+        """
+        :param val: string representing metric value
+        :param metric_type: string or numeric
+        :return: value converted to float if metric_type isn't string
+        """
+        if metric_type == 'string':
+            return val
+
+        try:
+            return json.loads(val)
```

Comment:
We need to ensure compatibility with Zabbix. This is a core script and has the possibility to cause serious issues if these changes are incompatible with zabbix.

So, I'd like to see the following things tested:

1) send an int to a zabbix item that is also an int and ensure the value makes it into zabbix

2) send a float to a zabbix item that is also a float and ensure the value makes it into zabbix

3) send a string to zabbix item that is also a string and ensure the value makes it into zabbix

This should be testable with either the zaio or the new cluster up stuff (it's been merged and the [local development doc ](https://github.com/openshift/openshift-tools/blob/prod/docs/local_development_monitoring.adoc) shows how to use it).